client/analytics/report/downloads/config.js

```diff
@@ -52,7 +52,7 @@ export const advancedFilters = applyFilters(
 		filters: {
 			product: {
 				labels: {
-					add: __( 'Product', 'woocommerce-admin' ),
+					add: __( 'Product Title', 'woocommerce-admin' ),
```

Comment:
I think "Title" would be misleading because the search is for actual products not strings found in the title.

![Screen Shot 2020-03-09 at 1 12 53 PM](https://user-images.githubusercontent.com/1922453/76173838-e2590780-6207-11ea-9dd1-bf60b811eecf.png)

The Orders Report Advanced Filter for Products is configured the same way.

![Screen Shot 2020-03-09 at 1 15 09 PM](https://user-images.githubusercontent.com/1922453/76173864-23e9b280-6208-11ea-845c-b61abb904d7b.png)

Reply:
Cool, I can delete that one again.

Reply:
Done.